service/src/main/java/com/codeforcommunity/processor/MapProcessorImpl.java

```diff
@@ -171,6 +176,42 @@ public SiteGeoResponse getSiteGeoJson() {
     if (!SiteGeoResponseCache.isExpired()) {
       return SiteGeoResponseCache.getResponse();
     }
+
+    Field<Timestamp> maxDate = max(SITE_ENTRIES.UPDATED_AT).as("maxDate");
+
+    Table<
```

Comment:
Is it just me or is the formatting for this really odd?

Have you run `mvn spotless:apply`?

Reply:
Yeah I ran it, idk why the formatting is so funky